svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java

```diff
@@ -224,8 +231,8 @@ private <T extends BipPayloadResponse> T makeRequest(
    * @return true if the API responds with OK status and response.
    */
   public boolean isApiFunctioning() {
```

Comment:
I'd say this method is unnecessary and can be removed. It's only used in test.

Reply:
Removal of isApiFunctioning would require a lot more significant changes, and since i didn't originate that function and there are unit tests, I'm going to leave it alone.  The one thing I did do was modified its implementation, so I'll go ahead and modify it back to what it was.

Reply:
Actually, I am recalling now that this function isApiFunctioning was added to allow for quick test cycles of checking the mutual TLS authentication with the BIP api. While it is "only" used in unit tests, we were ultimately able to get this test running on a windows based citrix environment.  It still doesn't seem feasible to get more than this one test running in a citrix environment, so that makes the isApiFunctioning unit tests still a valuable troubleshooting prospect.